chart/openfaas/values.yaml

```diff
@@ -24,7 +24,7 @@ faasnetesd:
     periodSeconds: 10
 
 gateway:
-  image: openfaas/gateway:0.9.14
+  image: openfaas/gateway:0.10.2
```

Comment:
Have you updated ./yaml/ too?

All changes to the charts need a bump in the chart version too.